deploy/kubernetes/helm-charts/altinn-repositories/values.yaml

```diff
@@ -6,9 +6,9 @@ image:
   repositoryStaging: altinntjenestercontainerregistry.azurecr.io/altinn-repositories-staging
   tag: latest
   pullPolicy: Always
-  containerPorts:
-    port: 3000
+  containerPort: 3000
```

Comment:
This limits us to only run one port per deployment/pod. A thing to keep in mind when we are moving forward.

Reply:
Keeping simple for now. Traefik 2 later.